apps/avatax/integration/setup.ts

```diff
@@ -0,0 +1,9 @@
+import { beforeAll } from "vitest";
+import { request, stash } from "pactum";
+import "./data/functions";
+
+beforeAll(() => {
+  request.setBaseUrl("https://shopex-avatax-318.eu.saleor.cloud");
```

Comment:
This will be passed from env....

So, I assume when we run parallel tests (eg 6 different saleorApiUrl), we will run the same test script in isolated containers, with own env set. 

Correct?

Reply:
I've changed it to use env variable. As you wrote this approach is much better for using GitHub Actions [matrix feature](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs) for running multiple jobs each for different Saleor environment.

We still need to figure out how to do that for entire monorepo (the jobs are defined at repo level, not project level): https://linear.app/saleor/issue/SHOPX-269/create-cicd-setup-for-running-tests-in-multiple-saleor-instances